src/getHumanAge.test.js

```diff
@@ -1,10 +1,62 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array`, () => {
+    expect(getHumanAge())
+      .toBeInstanceOf(Array);
+  });
+
+  it(`should return an array with 2 elements`, () => {
+    expect(getHumanAge(0, 0))
+      .toHaveLength(2);
+  });
+
+  it(`should return an array of 2 zero
+   if catAge & dogAge less than 15`, () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  it(`should return an array of 2 units
+   if catAge & dogAge equal to 15`, () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  it(`should return an array of 2 units
+   if catAge & dogAge less than 24`, () => {
+    expect(getHumanAge(23, 23))
+      .toEqual([1, 1]);
+  });
+
+  it(`should return an array of 2 deuces
+   if catAge & dogAge equal to 24`, () => {
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  it(`should return an array of 2 deuces
+   if catAge less than 28 & dogAge less than 29`, () => {
+    expect(getHumanAge(27, 28))
+      .toEqual([2, 2]);
+  });
+
+  it(`should return an array of 2 deuces
+   if catAge equal to 28 & dogAge equal to 29`, () => {
+    expect(getHumanAge(28, 29))
+      .toEqual([3, 3]);
+  });
+
+  it(`should return an array with elements 6 & 5
```

Comment:
```suggestion
  it(`should return an array with elements [6, 5]
```

if it's an array, better write like this

Reply:
Thanks. This mistake was already edited in all code.